dev/2017/spring/alumni-chapters-and-clubs/alumni-chapters-and-clubs.html

```diff
@@ -0,0 +1,66 @@
+<article class="story" id="knights-kingdom">
+  <div class="container-wide story-header-image" [background-image filename='alumni-map']>
+    <div class="container">
+      <div class="col-sm-6">
+        <h1>Knights&rsquo; Kingdom</h1>
+      </div>
+    </div>
+  </div>
+  <section id="intro">
+    <div class="container">
+      <div class="row">
+        <div class="col-md-8 col-md-offset-2">
+          <p>UCF alumni chapters and clubs are a great way to stay connected to everything you love about being a Knight. From prime networking events to fun tailgating parties, rewarding volunteer opportunities and much more, the experience of being an active member in your local group can bring benefits far beyond your degree.</p>
+          <p>&ldquo;With an alumni base of more than 263,000, it’s important to harness our scale and strengthen UCF&rsquo;s connection to our alumni through our chapters and clubs program,&rdquo; says Julie Stroh, senior associate vice president for advancement, alumni engagement and annual giving. &ldquo;Our regional groups are an integral part to helping us build a culture of service and philanthropy in support of our beloved university. Although we are in the very early stages of our international outreach, I can&rsquo;t wait to see what the future has in store as we broaden UCF’s global footprint.&rdquo;</p>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section id="questions">
+    <div class="well well-ucf-gold">
```

Comment:
The horizontal padding of .well here makes the content a bit narrow at the -xs breakpoint--maybe use a callout here instead?